Makefile

```diff
@@ -139,13 +140,7 @@ endif
 
 TARGET_ARCH_DEBIAN = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t debian)
 
-ifeq ($(RELEASE_TYPE),production)
-	PRODUCT_NAME = etcher
-endif
-ifeq ($(RELEASE_TYPE),snapshot)
-	PRODUCT_NAME = etcher-snapshots
-endif
-
+PRODUCT_NAME = etcher
```

Comment:
So I guess the longer `APPLICATION_VERSION` is now the only differentiation between locally-built `snapshot` and `production` builds?